data/README.md

```diff
@@ -1,10 +1,194 @@
+```
+- Loinc num: A unique identifier for a specific test or observation, typically in a 5-digit-then-a-dash format (e.g., 806-0).
+- Description: The description pulled for the 'Component Core' Part for correlating LOINC codes/concept.
+
+#### Extracts:
+
+- [LOINC Codes with Part Descriptions](./snoinc_extracts/loinc_codes_with_part_descriptions.csv)
+
+_**NOTE: we can easily change this to be a file with any delimiter instead of a `,`**_
+
+---
+
+### LOINC UMLS Related Names
+
+This data file organizes correlated terms from LOINC and other terminology sets, such as SNOMED, that correlate to a single LOINC code.  The UMLS `Atom` and `Crosswalk` APIs are leveraged to gather and organize this data.
+
+#### Data Structure:
+
+```json
+{
+   ...
+    "Epidermal Allergen Mix (Dog dander+Cat epithelium+Horse dander) Ab.IgE panel - Serum or Plasma": {
+        "code": "102115-3",
+        "names": [
+            "(Dog dander+Cat epithelium+Horse dander) Antibody.immunoglobulin E panel:-:To identify measures at a point in time:Serum/Plasma:-",
+            "Epid Allerg Mix IgE pl SerPl",
+            "(Dog dander+Cat epithelium+Horse dander) IgE pl",
+            "(Dog dander+Cat epithelium+Horse dander) Ab.IgE panel:-:Pt:Ser/Plas:-"
+        ]
+    },
+    ...
+}
+```
+- Key: LOINC Full Common Name
+- Code: A unique identifier for a specific test or observation, typically in a 5-digit-then-a-dash format (e.g., 806-0).
+- Names: A list of all related terms/names from the `atom` and `crosswalk` APIs for the LOINC Code.
+
+#### Extracts
+
+- [Loinc UMLS Related Names](./snoinc_extracts/loinc_umls_related_names_20250929.json)
+
+---
+
+### SNOMED
+
+These data files are for the various codes/concepts in SNOMED used to be the base of the TTC model.
+
+#### Data Structure:
+
+ ```csv
+ code|text
+442779003|Borderline low
+281301001|Within reference range
+ ```
+- Code: These are unique numerical identifiers for clinical concepts, such as a specific disease, a symptom, or a procedure.
+- Text: Each concept code is associated with one or more textual descriptions that human-readable terms for the concept. A concept can have several descriptions, including synonyms, which represent the same clinical idea.  For this data file there will just be a single text, the common name/term/description, associated with each code.
+
+#### Extracts:
+
+- [Lab Values](./snoinc_extracts/snomed_lab_value_20250926.csv) - SNOMED CT does not code the specific quantitative values of lab results (e.g., "glucose 105 mg/dL") but rather provides codes for the qualitative interpretation of a result (e.g., positive, negative, abnormal). The quantitative value and its units are typically stored separately in the health record. 
+
+---
+
+### HL7
+
+These data files are for the various codes & displays from various HL7 ValueSets and CodeSystems used in the base of the TTC model.
```

Comment:
I think we should just say "used in the [TTC model for X data element]" in all the places where you have this text.